examples/geospatial/plot_lines.py

```diff
@@ -0,0 +1,41 @@
+"""
+=====
+Graphs from a set of lines
+=====
+
+This example shows how to build a graph from a set of geographic lines 
+(sometimes called "linestrings") using PySAL and geopandas. We'll plot
+some streets in Tempe, Arizona, as well as the graph formed from the segments.  
+"""
+
+from libpysal import weights, examples
+from contextily import add_basemap
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy
+import geopandas
+
+# read in example data online
+filepath = examples.get_path("streets.shp")
+tempe = geopandas.read_file(filepath).to_crs(epsg=3857)
+
+coordinates = numpy.column_stack((tempe.centroid.x, tempe.centroid.y))
+
+# construct the graph
+w_graph = weights.Queen.from_dataframe(tempe)
+graph = w_graph.to_networkx()
```

Comment:
This should move down below the next heading, and bump `positions` down further still.